src/ProjectService.Models.Dto/Responses/RolesResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿using LT.DigitalOffice.ProjectService.Models.Dto.ResponsesModels;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Responses
+{
+    public class RolesResponse
+    {
+        public IEnumerable<Role> Roles { get; set; }
+        public int TotalCount { get; set; }
```

Comment:
Why is the total number needed?

Reply:
It is needed for the frontend to make pagination (to make sure how many roles they can show and skip)